### PR TITLE
Correct target filter in Scrap Trawler

### PR DESCRIPTION
**Pull request by Fulmene**

### CODE REVIEW

**Comment by melvinzhang** [comment]
@Fulmene is the bug is using `less than or equals` instead of `less than`?

**Comment by melvinzhang** [comment]
If so, please fix it by adding a cmcLT method to MagicCardFilterImpl

File: release/Magarena/scripts/Scrap_Trawler.groovy [hunk 1]
```diff
@@ -10,7 +24,7 @@
             return new MagicEvent(
                 permanent,
                 new MagicTargetChoice(
-                    card(MagicType.Artifact).cmcLEQ(cmc).from(MagicTargetType.Graveyard),
```
Comment (melvinzhang) [inline release/Magarena/scripts/Scrap_Trawler.groovy]: change cmcLEQ to cmcLT, this is clearer and easier to read. Will need to add cmcLT to MagicCardFilterImpl first.

**Comment by Fulmene** [comment]
Understood. I actually thought so but was unsure.

**Comment by melvinzhang** [comment]
Thanks for the improvements. Generally we want groovy code to be simple and prefer the complexity to be into the Java code. The same Java code can also be used by several different groovy code, so overall it is less complex.

**Pull request merged**